app/pages/project/index.cjsx

```diff
@@ -211,6 +219,13 @@ ProjectPageController = React.createClass
       owner: null
       preferences: null
 
+  fetchProject: (owner, name) ->
+    @setState
+      loading: true
+      error: null
+      project: null
+      owner: null
+
```

Comment:
The added lines 222-228 should be removed. It's breaking the project from loading. Possibly mistake in rebase?